### ISSUE TITLE
为什么我的train出来效果很差？

### ISSUE THREAD

**Issue by pinkie14104** [undersail/easybot#3]
我用的原版来跑的，https://github.com/llSourcell/tensorflow_chatbot。跑出来效果好差啊，不成句子啊。请问您训练到什么程度啊？我的跑了一周了，跑到step43500，其中ppl到5.8了都。

**Comment by undersail** [comment]
global step 91800 learning rate 0.3413 step-time 2.43 perplexity 2.65

**Comment by pinkie14104** [comment]
hi你好，看到您的评论，我也尝试多跑了些step,但是还是达不到您跑出来的效果。请问您做过其它的调参吗？
